spark/v3.4/build.gradle

```diff
@@ -75,7 +75,7 @@ project(":iceberg-spark:iceberg-spark-${sparkMajorVersion}_${scalaVersion}") {
       exclude group: 'org.roaringbitmap'
     }
 
-    compileOnly "org.apache.datafusion:comet-spark-spark${sparkMajorVersion}_${scalaVersion}:0.5.0"
+    compileOnly "org.apache.datafusion:comet-spark-spark${sparkMajorVersion}_${scalaVersion}:0.8.1"
```

Comment:
is this something that we should define in `libs.versions.toml`? Then dependabot would take care of bumping the version whenever there is a new release

Reply:
I moved Comet version to `libs.versions.toml`. Thanks!